Gum/Plugins/InternalPlugins/Behaviors/MainBehaviorsPlugin.cs

```diff
@@ -219,7 +219,7 @@ private void UpdateTabPresence()
         {
             viewModel.UpdateTo(asComponent);
 
-            this.behaviorsTab.Show(focus: false);
+            this.behaviorsTab.Show(false);
```

Comment:
"magic bool" params should either have local variables or parameter (like `focus:`) to explain their usage.